tests/foreman/ui/test_activationkey.py

```diff
@@ -320,7 +320,7 @@ def test_negative_create_with_invalid_limit(self):
                         limit=limit,
                     )
                     self.assertIsNotNone(self.activationkey.wait_until_element(
-                        locators['ak.invalid_limit']))
+                        locators['hostcollection.invalid_limit']))
```

Comment:
yes, but content hosts from that perspective is more functionality of host collections rather than activation keys. It doesn't make sense to duplicate locators, just for logical reasons